pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -311,20 +311,32 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 	return managedresources.CreateForSeed(ctx, b.client, b.namespace, managedResourceControlName, false, resources)
 }
 
-func getDruidDeployCommands(config *config.GardenletConfiguration) []string {
+func getDruidDeployCommands(gardenletConf *config.GardenletConfiguration) []string {
 	command := []string{"" + "/bin/etcd-druid"}
 	command = append(command, "--enable-leader-election=true")
 	command = append(command, "--ignore-operation-annotation=false")
-	command = append(command, "--workers=50")
-	command = append(command, "--compaction-workers=3")
+	if gardenletConf == nil {
+		return command
+	}
+
+	config := gardenletConf.ETCDConfig
+	if config == nil {
+		return nil
+	}
+	if config.ETCDController != nil {
+		command = append(command, "--workers="+strconv.FormatInt(pointer.Int64Deref(config.ETCDController.Workers, 3), 10))
```

Comment:
Hm, before this change we set `--workers=50` and now it's default value is only `3`. I think this can be very dangerous if one doesn't pay intention and now sets the value via the component config. Can you make a default of `50` again?